themes/answers-hitchhiker-theme/.github/CODEOWNERS

```diff
@@ -0,0 +1 @@
+* @yext/watson
```

Comment:
I don't think this would work since they are under different organizations (yext vs yext-pages)

Reply:
Oh I missed that this was added as a change. Do you think we need to delete it?